pkg/tracer/common/common_linux.go

```diff
@@ -3,15 +3,18 @@
 package common
 
 import (
-	bpflib "github.com/iovisor/gobpf/elf"
 	"os"
+
+	bpflib "github.com/iovisor/gobpf/elf"
 )
 
 const (
 	V4UDPMapName           = "udp_stats_ipv4"
 	V6UDPMapName           = "udp_stats_ipv6"
 	V4TCPMapName           = "tcp_stats_ipv4"
 	V6TCPMapName           = "tcp_stats_ipv6"
+	ActiveFdsMapName       = "active_fds"
```

Comment:
I think all the Go code relating to active fd's can be removed. That was only added for intermediate testing afaik.